packages/utils/src/phoneNumbers.ts

```diff
@@ -231,6 +231,34 @@ export function anonymizedPhone(phoneNumber: string) {
   return phoneNumber.slice(0, -4) + 'XXXX'
 }
 
+export function getExampleNumber(
+  regionCode: string,
+  showZero: boolean = true,
+  isInternational: boolean = false
+) {
+  let examplePhone =
+    phoneUtil.getExampleNumber(regionCode) ||
+    phoneUtil.getExampleNumber(getRegionCodeFromCountryCode(regionCode) as string)
```

Comment:
Why do we need this line?

Reply:
To accept the country code and the region code (`+54` or `DE`).

Reply:
Right but why do we need to accept either one? Can't we consistently pass in just one type?

Reply:
It's ok for me :+1: